modules/core/src/main/java/org/apache/ignite/IgniteCache.java

```diff
@@ -1617,20 +1616,6 @@ public <T> IgniteFuture<Map<K, EntryProcessorResult<T>>> invokeAllAsync(Set<? ex
      */
     public CacheMetrics localMetrics();
 
-    /**
-     * Gets whole cluster MxBean for this cache.
-     *
-     * @return MxBean.
-     */
-    public CacheMetricsMXBean mxBean();
```

Comment:
It seems we can keep this method, but makes it to return `CacheStatisticsMXBean` as required by JCache specification.

Reply:
JCache requires registration of these mx bean by calling CacheManager#enableStatistics(), CacheManager#enableManagement() for example. It doesn't require Cache#mxBean(). Otherwise, we'd have to override the method and would'nt pass the jcache-tck tests. We remove statistics mx beans here. I suggest to hide any related interfaces and calls. WDYT?